Reference_Manual.md

```diff
@@ -1260,6 +1261,7 @@ Note that the server needs to be configured to use the same username and passwor
                 </identity>
             </realm>
         </realms>
+        <member-authentication realm="usernamePasswordIdentityRealm"/>
```

Comment:
OK, I just looked at the code and as we discussed, we should be using the `client-authentication` here. The problem was that the use of `identity` configuration was wrong. It looks like the server expects you to use `authentication` instead. So, the correct member config should be like

```xml
    <security enabled="true">
        <realms>
            <realm name="usernamePasswordIdentityRealm">
                <authentication>
                    <simple>
                        <user username="test-user" password="test-pass"/>
                    </simple>
                </authentication>
            </realm>
        </realms>
        <client-authentication realm="usernamePasswordIdentityRealm"/>
    </security>
```

Can you try this and update the config with it?